src/data_provider/src/sysInfoWin.cpp

```diff
@@ -173,9 +173,8 @@ class SysInfoProcess final
 
             for (const auto& logicalDrive : logicalDrives)
             {
-                const auto normalizedName { logicalDrive.back() == L'\\' ? logicalDrive.substr(0, logicalDrive.length() - 1) : logicalDrive };
                 const auto spDosDevice { std::make_unique<char[]>(OS_MAXSTR) };
-                res = QueryDosDevice(normalizedName.c_str(), spDosDevice.get(), OS_MAXSTR);
+                res = QueryDosDevice(logicalDrive.c_str(), spDosDevice.get(), OS_MAXSTR);
```

Comment:
Be carefull with this, it is reverting the fix: https://github.com/wazuh/wazuh/issues/8329

Reply:
Thank you for notifying, the change sneaked into my local branch.